alkira/resource_alkira_connector_gcp_interconnect.go

```diff
@@ -0,0 +1,340 @@
+package alkira
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraConnectorGcpInterconnect() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage GCP Interconnect.",
```

Comment:
I will fix some documentation after this initial commit. It's not point to chase every single place.